pkg/service/bindings.go

```diff
@@ -396,11 +396,16 @@ func (sb *ServiceBindings) addServiceTarget(name string, service string, port ma
 		identifier: service,
 		namespace:  namespace,
 	}
+	targetService := service
+	// on annotated services, namespace is not yet available
+	if namespace != "" {
+		targetService = fmt.Sprintf("%s.%s", service, namespace)
```

Comment:
As namespace cannot be specified when target is service, we don't need this I don't believe.

Reply:
It is no longer needed.